streamlit_folium/frontend/src/index.tsx

```diff
@@ -269,17 +269,27 @@ function onRender(event: Event): void {
     }
   }
 
+  var view_changed = false
+  var new_zoom = window.map.getZoom()
```

Comment:
Tiny nit, could you just make this and new_center `null`, since this default value is never actually used? You could also then just check if either one is not null, rather than needing `view_changed`.

Reply:
@blackary 
Hum, not sure it would work. 
e.g. If the center is updated but the zoom stays the same, then we need to get the current map zoom for the setView call line 292 (to keep the same zoom).

Reply:
🤦 Yup, you're right -- good call